src/deviceApiCalls/transports/extension.transport.js

```diff
@@ -73,3 +89,16 @@ async function getContentScopeConfig () {
         )
     })
 }
+
+async function getIncontextSignupDismissedAt () {
+    return new Promise(resolve => {
+        chrome.runtime.sendMessage(
```

Comment:
Same as the other comment about outgoing messages, if this is just about proving out the extension flow then I don't see an issue with adding these messages here - but if we expect/intend to share this functionality then we should follow up with moving this to be schema-based etc